### PR TITLE
LPS-133843 Add condition to Kaleo Forms to fix permission problem

### PR DESCRIPTION
**Pull request by MariMalta**

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/kaleo_draft_definition_action.jsp [hunk 1]
```diff
@@ -51,7 +53,9 @@ KaleoDefinitionVersion kaleoDefinitionVersion = (KaleoDefinitionVersion)row.getO
 		<portlet:param name="name" value="<%= kaleoDefinitionVersion.getName() %>" />
 	</portlet:actionURL>
 
-	<liferay-ui:icon-delete
-		url="<%= deleteURL %>"
-	/>
+	<c:if test="<%= permissionChecker.isCompanyAdmin() %>">
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/kaleo_draft_definition_action.jsp:56]: Hey @MariMalta, please move the `portlet:actionURL` declaration to the if statement as well.

Thx

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/kaleo_draft_definition_action.jsp [hunk 2]
```diff
@@ -38,11 +38,13 @@ KaleoDefinitionVersion kaleoDefinitionVersion = (KaleoDefinitionVersion)row.getO
 		<portlet:param name="draftVersion" value="<%= kaleoDefinitionVersion.getVersion() %>" />
 	</liferay-portlet:renderURL>
 
-	<liferay-ui:icon
-		message="edit"
-		onClick='<%= "javascript:" + liferayPortletResponse.getNamespace() + "editWorkflow('" + editURL + "');" %>'
-		url="javascript:;"
-	/>
+	<c:if test="<%= permissionChecker.isCompanyAdmin() %>">
+		<liferay-ui:icon
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/kaleo_draft_definition_action.jsp:42]: Hey @MariMalta, please move the `liferay-portlet:renderURL` declaration to the if statement as well.

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MariMalta** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**